docs/sphinx/source/authenticating-to-vespa-cloud.ipynb

```diff
@@ -0,0 +1,984 @@
+    "## Control-plane vs Data-plane\n",
+    "\n",
+    "This may be self-explanatory for some, but it is worth mentioning that Vespa Cloud has two main components: the control-plane and the data-plane, which provide access to different functionalities.\n",
+    "\n",
+    "|                                                                                              | Control-plane | Data-plane | Comments                                                                                   |\n",
+    "| -------------------------------------------------------------------------------------------- | ------------- | ---------- | ------------------------------------------------------------------------------------------ |\n",
+    "| Deploy application                                                                           | ✅            | ❌         |                                                                                            |\n",
+    "| Modify application (re-deploy)                                                               | ✅            | ❌         |                                                                                            |\n",
+    "| Add or modify data-plane certs or token(s)                                                   | ✅            | ❌         |                                                                                            |\n",
+    "| Feed data                                                                                    | ❌            | ✅         |                                                                                            |\n",
+    "| Query data                                                                                   | ❌            | ✅         |                                                                                            |\n",
+    "| Delete data                                                                                  | ❌            | ✅         |                                                                                            |\n",
+    "| [Visiting](https://docs.vespa.ai/en/visiting.html)                                           | ❌            | ✅         |                                                                                            |\n",
+    "| [Monitoring](https://cloud.vespa.ai/en/monitoring)                                           | ❌            | ✅         |                                                                                            |\n",
+    "| Get application package                                                                      | ✅            | ❌         |                                                                                            |\n",
+    "| [`vespa auth login`](https://docs.vespa.ai/en/reference/vespa-cli/vespa_auth_login.html)     | ✅            | ❌         | Interactive control-plane login in browser                                                 |\n",
```

Comment:
these links did not display well for me, maybe the local processing ...

Reply:
Tried to build HTML, and indeed, they did not display well. 
Fixed by removing the `pre`-tags